analytics/analytics/job_processor/build_timings.py

```diff
@@ -103,7 +103,14 @@ def get_package_mapping(timings: list[dict]) -> dict[str, PackageDimension]:
 
 
 def create_build_timing_facts(job_fact: JobFact, gljob: ProjectJob):
-    timings = [t for t in get_timings_json(gljob) if t.get("name") and t.get("hash")]
+    # Sometimes the timing file isn't present. This could be due to a number of reasons;
+    # since `install_times.json` is created using `spack` itself, there's no guarantee
+    # that the file will be present.
+    try:
+        data = get_timings_json(gljob)
+        timings = [t for t in data if t.get("name") and t.get("hash")]
+    except JobArtifactFileNotFound:
```

Comment:
Just to be clear, we _don't_ want to catch `JobArtifactDownloadFailed` here, right? As in, we want to consider missing all artifacts a failure, but missing only the timings file isn't a failure?

Reply:
Right. The only time a job should be missing artifacts is when it failed. If a successful job doesn't have any artifacts, I think a sentry alert would be appropriate